### DISCUSSION TITLE
how can it debug fronted

### DISCUSSION THREAD

**Discussion by hellojukay** [wailsapp/wails#1204] [Q&A] [answered]
1. how can it debug fronted?
2. where is the fronted logs?
3. does chrome devtools works?

**Comment by alanchenchen** [comment, marked as answer]
1. context right click will open weview2 devtool on windows.
2. FE logs show at the console tab of devtool
3. maybe not..

**Reply by hellojukay** [reply]
thanks,it works

**Reply by alanchenchen** [reply]
There are still so many api docs confusing. While you build the app, devtool is hidden, it's not  as convenient as Electron.

**Reply by hellojukay** [reply]
i allways build with `wails build` not `wails dev` , so devtool was hidden.
